### DISCUSSION TITLE
⏳ Keep Elementor settings clean and focused in one section (settings).

### DISCUSSION THREAD

**Discussion by 79ho3ein** [elementor/elementor#24280] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team,

In Elementor, there are various settings that are scattered in different sections. For example, we now have these sections :

- settings
- tools
- element manager
- Role Manager

I don't think this is good. You can move all of these items to the settings section to make everything easier to access. 

### Describe the solution you'd like

The image below is something that would be great. All settings are accessible in one section. There is also a section dedicated to the Performance to contain any options that are related to the Performance. Like the CSS Print Method option and even Regenerate CSS & Data.

![settings](https://github.com/elementor/elementor/assets/77163248/fd443ee4-a168-436a-8912-0afcbd8e08e3)

In this case, everything is clean and easy to access.

**Comment by fabigrafo** [comment]
Yes, I would very much support that!

**Comment by fxaviers** [comment]
I appreciate that. Some settings are not used at all times, so I need to scroll through pages. A layout like this is extremely more simplified.

**Comment by fabigrafo** [comment]
Right now it is a mess. Really!
